common/app/model/dotcomrendering/MostPopular.scala

```diff
@@ -1,15 +1,51 @@
 package model.dotcomrendering
 
+import com.gu.contentapi.client.utils.format.{ArticleDesign, NewsPillar, StandardDisplay}
+import model.ContentFormat
 import play.api.libs.json._
 
+sealed trait OnwardsSource
+object OnwardsSource {
+  implicit val writes = Writes[OnwardsSource] {
+    case CuratedContent  => JsString("curated-content")
+    case MoreOnThisStory => JsString("more-on-this-story")
+    case Series          => JsString("series")
+    case PopularInTag    => JsString("popular-in-tag")
+    case RelatedStories  => JsString("related-stories")
+  }
+
+  case object CuratedContent extends OnwardsSource
+  case object MoreOnThisStory extends OnwardsSource
+  case object Series extends OnwardsSource
+  case object PopularInTag extends OnwardsSource
+  case object RelatedStories extends OnwardsSource
+}
+
+trait OnwardsCollection {
+  val heading: String
+  val onwardsSource: OnwardsSource
+  val format: ContentFormat
+}
 case class OnwardCollectionResponse(
     heading: String,
     trails: Seq[Trail],
-)
+    onwardsSource: OnwardsSource,
```

Comment:
This change is breaking other places where `OnwardCollectionResponse` is used, e.g. `storyPackage`:
https://github.com/guardian/frontend/blob/main/common/app/model/dotcomrendering/DotcomRenderingUtils.scala#L302-L304
https://github.com/guardian/frontend/blob/main/common/app/model/dotcomrendering/DotcomRenderingDataModel.scala#L81

Does it make sense to make `onwardsSource` and `format` optional? I believe this would need DCR changes to handle them but not sure

Reply:
DCR is already expecting those in the `/Onwards` endpoint: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/onwards.ts#L6-L13. It's the `storyPackage` in `CAPIArticleType` that's not expecting them: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/frontend.ts#L71-L74.

Reply:
[`DeeplyReadAgent`](https://github.com/guardian/frontend/blob/main/common/app/agents/DeeplyReadAgent.scala#L9-L11) is also affected as `mostPopular` is a `Seq[OnwardsCollectionResponse]` in the [model](https://github.com/guardian/frontend/blob/main/common/app/model/dotcomrendering/DotcomRenderingDataModel.scala#L100). This might not be a problem though because it is [currently under development](https://github.com/guardian/frontend/compare/main...deeplyreadlifecycle) and things have changed. I also suspect this change is expected. But ccing @jlieb10 and @mxdvl to be on the safe side.